### PR TITLE
login fixed

### PR DESCRIPTION
**Pull request by Miscellian**

### CODE REVIEW

File: GameSalesApi/GameSalesApi/Features/Authorization/CommandHandlers/NewUserCommandHandler.cs [hunk 1]
```diff
@@ -50,10 +51,10 @@ public override Result Handle(NewUserCommand input)
             string passwordHash = PasswordHelpers.HashPassword(input.Password, passwordSalt);
 
             User newUser = new User()
-            {   
+            {
                 Id = Guid.NewGuid(),
                 Email = input.Email,
-                PasswordSalt = passwordSalt.ToString(),
+                PasswordSalt = Convert.ToBase64String(passwordSalt),
```
Comment (Anatolii-Ivanov) [inline GameSalesApi/GameSalesApi/Features/Authorization/CommandHandlers/NewUserCommandHandler.cs:56]: Why? And why you don't convert it on login back??

File: GameSalesApi/GameSalesApi/Helpers/PasswordHelpers.cs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Security.Cryptography;
+using System.Text;
```
Comment (Anatolii-Ivanov) [inline GameSalesApi/GameSalesApi/Helpers/PasswordHelpers.cs]: ?

File: game-sales-tracker/src/app/authorization-module/login-form/login-form.component.ts [hunk 3]
```diff
@@ -36,7 +40,13 @@ export class LoginFormComponent implements OnInit {
     }
 
     public onSubmit(): void {
-        this.credentials = this.loginForm.value;
-        this.loginFormDirective.resetForm();
+        this.authService.login(this.loginForm).subscribe(
+            () => this.router.navigate(["/"]).then(
+                this.loginFormDirective.resetForm()),
```
Comment (Anatolii-Ivanov) [inline game-sales-tracker/src/app/authorization-module/login-form/login-form.component.ts]: Why not?
   () => {
        this.router.navigate(["/"]);
        this.loginFormDirective.resetForm();
}

File: game-sales-tracker/src/app/authorization-module/register-form/register-form.component.ts [hunk 4]
```diff
@@ -71,12 +71,12 @@ export class RegisterFormComponent implements OnInit {
     public onSubmit(): void {
         this.authService.register(this.registerForm)
             .subscribe(
-                () => this.router.navigate(["/login"]),
+                () => this.router.navigate(["/login"]).then(
```
Comment (Anatolii-Ivanov) [inline game-sales-tracker/src/app/authorization-module/register-form/register-form.component.ts]: The same

File: game-sales-tracker/src/app/landing-page/landing-page.component.ts [hunk 5]
```diff
@@ -1,27 +1,10 @@
...
+import { Component } from "@angular/core";
 
 @Component({
-  selector: 'app-landing-page',
-  templateUrl: './landing-page.component.html',
-  styleUrls: ['./landing-page.component.css']
+    selector: "app-landing-page",
+    templateUrl: "./landing-page.component.html",
+    styleUrls: ["./landing-page.component.css"],
 })
```
Comment (Anatolii-Ivanov) [inline game-sales-tracker/src/app/landing-page/landing-page.component.ts:7]: Do we use this component somewhere?

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]

File: game-sales-tracker/src/app/authorization-module/login-form/login-form.component.ts (hunk 3 above)
Comment (Miscellian) [reply game-sales-tracker/src/app/authorization-module/login-form/login-form.component.ts]: Why not?
() => this.router.navigate(["/"]).then(
                this.loginFormDirective.resetForm()),

File: game-sales-tracker/src/app/landing-page/landing-page.component.ts (hunk 5 above)
Comment (Miscellian) [reply game-sales-tracker/src/app/landing-page/landing-page.component.ts:7]: +

**Pull request merged**